### DISCUSSION TITLE
Having issues installing with conda

### DISCUSSION THREAD

**Discussion by EvanZ** [Lightning-AI/pytorch-lightning#14192] [General]
I'm using a Deep Learning AMI on EC2. When I try to `conda install pytorch-lightning` it gives me the following error:

```
(pytorch) [ec2-user@ip-172-18-84-254]$ conda install pytorch-lightning
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are missing from the target environment:
  - magma-cuda116==2.6
  - fastai==2.1
  - cmake==3.22
```

Does conda not install automatically the required packages? I'm not sure what to do here.

**Comment by awaelchli** [comment]
Hi
You need to add `-c conda-forge`.
Be aware, there seem to be some installation issues with the conda package (#14765). 
You can also do `pip install pytorch-lightning` in the conda environment.

**Comment by lovelyweather** [comment]
Have you solved yet? I am in same situation as you. Using Deep Learning AMI on EC2, when I try to use `conda install pytorch-lightning`,  it gives the same error.
`PackagesNotFoundError: The following packages are missing from the target environment:
  - magma-cuda116==2.6
  - cmake==3.22`
 
Using the `Conda list`, its has both the packages.
```
magma-cuda116             2.6.1                         1    pytorch
cmake                     3.24.1.1                 pypi_0    pypi
```
@EvanZ

**Comment by GADV** [comment]
Has anybody solved this yet? I am in a similar situation.
When I try to install transformers (from huggingface) I get the same error: packages are in the conda list but conda does not see them (Ubuntu,  Deep Learning AMI on EC2).
